content/docs/guides/RBAC.md

```diff
@@ -0,0 +1,215 @@
++++
+title = "Role based access control"
+description = "Implementing RBAC with Biscuit"
+date = 2021-05-01T08:00:00+00:00
+updated = 2021-05-01T08:00:00+00:00
+draft = false
+weight = 10
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+lead = "Implementing RBAC with Biscuit"
+toc = false
+top = false
++++
+
+Role-based access control is a common authorization model where a set of permissions is assigned to a role, and a user or program can have one or more roles.
+This makes permissions more manageable than giving them to users directly: a role can be designed for a set of tasks, and can be given or taken back from the user depending on their duties, in one operation, instead of reviewing te user's entire set of rights. Changing the permissions of a role can also be done without going through all the users.
```

Comment:
```suggestion
This makes permissions more manageable than giving them to users directly: a role can be designed for a set of tasks, and can be given or taken back from the user depending on their duties, in one operation, instead of reviewing the user's entire set of rights. Changing the permissions of a role can also be done without going through all the users.
```